modules/end-user-guide/partials/proc_creating-workspaces.adoc

```diff
@@ -2,4 +2,121 @@
 [id="creating-workspaces"]
 = Creating workspaces
 
-Rough technical info to be dumped here by the assigned SME.
+If your use case does not permit use of the {prod-short} dashboard, you can create workspaces with {orch-name} APIs by applying custom resources to the cluster.
+
+[NOTE]
+====
+
+Creating new workspaces through the {prod-short} dashboard provides better user experience and configuration benefits compared to using the command line:
+
+* As a user, you are automatically logged in to the {orch-name} cluster.
+* {platforms-name} clients work automatically.
+* {prod-short} and its components automatically convert the target Git repository's devfile into the `DevWorkspace` and `DevWorkspaceTemplate` custom resources on the cluster.
+* Access to the workspace is secured by default with the `routingClass: che` in the `DevWorkspace` of the workspace.
+* Recognition of the `DevWorkspaceOperatorConfig` configuration is managed by {prod-short}.
+* Recognition of configurations in `spec.devEnvironments` specified in the `CheCluster` custom resource including:
+** Persistent storage strategy is specified with `devEnvironments.storage`.
+** Default IDE is specified with `devEnvironments.defaultEditor`.
+** Default plugins are specified with `devEnvironments.defaultPlugins`.
+** Container build configuration is specified with `devEnvironments.containerBuildConfiguration`.
+
+====
+
+.Prerequisites
+
+* An active `{orch-cli}` session with permissions to create `DevWorkspace` resources in your {orch-namespace} on the cluster. See {orch-cli-link}.
+
+* You know the relevant {prod-short} user namespace on the cluster.
++
+TIP: You can visit `pass:c,a,q[{prod-url}]/api/kubernetes/namespace` to get your {prod-short} user namespace as `name`.
+
+* You are in the {prod-short} user namespace on the cluster.
++
+NOTE: {prod-short} administrators who intend to create workspaces for other users must create the `DevWorkspace` custom resource in a user namespace that is provisioned by {prod-short} or by the administrator. See xref:administration-guide:configuring-namespace-provisioning.adoc[].
+
+.Procedure
+
+. To prepare the `DevWorkspace` custom resource, copy the contents of the target Git repository's devfile.
++
+.Copied devfile contents with `schemaVersion: 2.2.0`
+====
+[source,yaml,subs="+quotes,+attributes"]
+----
+components:
+  - name: tooling-container
+    container:
+      image: quay.io/devfile/universal-developer-image:ubi8-latest
+----
+====
++
+TIP: For more details, see the link:https://devfile.io/docs/2.2.0/what-is-a-devfile[devfile v2 documentation].
+
+. Create a `DevWorkspace` custom resource, pasting the devfile contents from the previous step under the `spec.template` field.
++
+.A `DevWorkspace` custom resource
+====
+[source,yaml,subs="+quotes,+attributes,+macros"]
+----
+kind: DevWorkspace
+apiVersion: workspace.devfile.io/v1alpha2
+metadata:
+  name: my-devworkspace#<1>
+  namespace: user1-dev#<2>
+spec:
+  routingClass: che
```

Comment:
I assume this is the same for both Che and Dev Spaces?

Reply:
Yes, is the same for Che and Dev Spaces